dev/reference/tools/error-case-handling/lightning-misuse-handling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Common Misuses of TiDB Lightning
+category: reference
+---
+
+# Common Misuses of TiDB Lightning
+
+This document introduces common error scenarios in using [TiDB Lightning](/dev/reference/tools/tidb-lightning/overview.md), and also provides their reasons and solutions.
+
+## Error: `checksum mismatched remote vs local`
+
+The following error is reported during data import:
+
+```log
+Error: checksum mismatched remote vs local => (checksum: 3828723015727756136 vs 7895534721177712659) (total_kvs: 1221416844 vs 1501500000) (total_bytes:237660308510 vs 292158203078)
+```
+
+### Reasons
+
+* TiDB Lightning was used to import data previously, and the corresponding [`checkpoint`](/dev/reference/tools/tidb-lightning/checkpoints.md) data was not cleaned up. To confirm this, you can check the log at the first launch of TiDB Lightning:
+
+    * When `[checkpoint] driver = file`, if the log that marks the beginning of data import using TiDB Lightning shows `open checkpoint file failed, going to create a new one`, then the `checkpoint` is cleaned correctly. Otherwise, the remaining data might lead to imported data missing. `[checkpoint] driver = file`.
```

Comment:
```suggestion
    * When `[checkpoint] driver = file`, if the log that marks the beginning of data import using TiDB Lightning shows `open checkpoint file failed, going to create a new one`, then the `checkpoint` is cleaned correctly. Otherwise, the remaining data might lead to imported data missing.
```